plasma/client/client.py

```diff
@@ -67,3 +76,20 @@ def get_block(self, blknum):
 
     def get_current_block_num(self):
         return self.child_chain.get_current_block_num()
+
+    def finalize_exits(self, account):
+        self.root_chain.finalizeExits(NULL_ADDRESS, transact={'from': account})
+
+    def challenge_exit(self, blknum, confirmSig, account):
+        oindex = 0
+        utxo_pos = encode_utxo_id(blknum, 0, oindex)
+
+        tx = self.get_transaction(blknum, 0)
+        tx_bytes = rlp.encode(tx, UnsignedTransaction)
+
+        merkle = FixedMerkle(16, [tx.merkle_hash], True)
```

Comment:
I think this will only work if the block only has a single transaction in it. We probably need to pull the entire block given `blknum` and generate the Merkle proof using that block.

Reply:
Ok, i'll change it later